### ISSUE TITLE
Ergonomics for naive::Days and Months

### ISSUE THREAD

**Issue by nathan** [chronotope/chrono#1208]
`naive::Days` and `Months` are extremely unergonomic at the moment; it's *impossible* to recover the number of days/months once one has been constructed, and the constructors `Days::new` and `Months::new` are rather verbose.

Would it be possible for `chrono` to provide `From` and `Into` impls for `u64`/`u32` (respectively) and/or make each integer `pub` so users can construct and destructure them?

**Comment by djc** [comment]
What's your use case for this? It seems unergonomic because you're using them in a way that we didn't foresee when designing them.

**Comment by djc** [comment]
Yes, they mostly exist for the `Add`/`Sub` impls. I don't want to expose the inner field publicly so you won't be able to write `Days(1)` directly.

**Comment by danwilliams** [comment]
@nathan I don't really follow about the constructors, but regarding `From`, [this PR](https://github.com/chronotope/chrono/pull/1373) adds `Months::as_u32()`, which should allow you to determine the value of months. Using that method you can then easily implement any `From` conversions you need 🙂

**Comment by djc** [comment]
Also I think all of this will be superseded by the `CalendarDuration` type as proposed in #1290.
